geti_sdk/http_session/geti_session.py

```diff
@@ -25,10 +25,10 @@
 from requests.structures import CaseInsensitiveDict
 from urllib3.exceptions import InsecureRequestWarning
 
-from geti_sdk.platform_versions import GETI_18_VERSION, GETI_114_VERSION, GetiVersion
+from geti_sdk.platform_versions import GETI_116_VERSION, GetiVersion
 
 from .exception import GetiRequestException
-from .server_config import LEGACY_API_VERSION, ServerCredentialConfig, ServerTokenConfig
+from .server_config import ServerCredentialConfig, ServerTokenConfig
 
 CSRF_COOKIE_NAME = "_oauth2_proxy_csrf"
```

Comment:
I still see that we have legacy code for authentication using user password and cookies, I would recommend creating a ticket for cleaning the authentication code and use the API key as the only valid approach for authentication.

Reply:
The username/password approach is kept in place mainly for testing purposes and ease of use for on-prem users. We are clear in the instructions that the API key is the recommended approach, though. 

I'd prefer to keep it around for a bit longer, but we could add a deprecation or security warning when user/pass is used for authentication. Would that be ok for you?

Reply:
Sounds good to me, thanks Ludo!

Reply:
implemented!